README.md

```diff
@@ -51,7 +50,7 @@ Firstly, to obtain a copy of the (Ci40) Marduk platform supported OpenWrt source
 
 Install build dependencies:
 
-    $ sudo apt-get install libncurses5-dev libncursesw5-dev zlib1g-dev libssl-dev gawk subversion device-tree-compiler
+    $ sudo apt-get install libncurses5-dev libncursesw5-dev zlib1g-dev libssl-dev gawk subversion device-tree-compiler bsdmainutils
```

Comment:
@Ham22 are these mandatory for building OpenWrt or just for the docker build environment?